src/lib/utils/SynapseClient.ts

```diff
@@ -2229,7 +2234,7 @@ export const getAccessRequirement = (
 export const getAccessRequirementStatus = (
   sessionToken: string | undefined,
   requirementId: string | number,
-): Promise<AccessRequirementStatus | ManagedACTAccessRequirementStatus> => {
+): Promise<AccessRequirementStatus> => {
   const url = `repo/v1/accessRequirement/${requirementId}/status`
```

Comment:
Same as above. This line is required for the ManagedACTAccess component.

Reply:
added this back in